setup.cfg

```diff
@@ -0,0 +1,31 @@
+[pycodestyle]
+max-line-length = 99
+
+[pylint.MASTER]
+jobs = 3
+max-line-length = 99
+fail-under=9.9
+
+load-plugins =
+  pylint.extensions.bad_builtin,
+  pylint.extensions.broad_try_clause,
+  pylint.extensions.check_elif,
+  pylint.extensions.comparetozero,
+  pylint.extensions.emptystring,
+  pylint.extensions.overlapping_exceptions
+
+[pylint.'MESSAGES CONTROL']
+disable=
+        ; docstring
```

Comment:
2 spaces instead of 8? Exactly like you did it above. Let's say it is more or less a standard for files like this 😉